Modules/TOF/include/TOF/TaskRaw.h

```diff
@@ -83,6 +83,7 @@ class RawDataDecoder final : public DecoderBase
   Counter<nequipments, nullptr> mCounterNoisyChannels;    /// Counter for noisy channels
   Counter<1024, nullptr> mCounterTimeBC;                  /// Counter for the Bunch Crossing Time
   Counter<91, nullptr> mCounterNoiseMap[ncrates][4];      /// Counter for the Noise Hit Map
```

Comment:
If you feel particularly majestic you could add in the comment that we have one per crate and one per FEA (the documentation will pay off in the future!)